test/end-to-end/emailverification.test.js

```diff
@@ -925,3 +925,81 @@ describe("Email verification signOut errors", function () {
         assert.strictEqual(await error.evaluate((e) => e.textContent), SOMETHING_WENT_WRONG_ERROR);
     });
 });
+
```

Comment:
add a test where we init web-js with custom datepicker and then check that that is being used.